cmake/pkgconfig/ignition.in

```diff
@@ -1,10 +1,10 @@
-prefix=${pcfiledir}/../..
-libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+prefix=${pcfiledir}/@PC_CONFIG_RELATIVE_PATH_TO_PREFIX@
+libdir=${prefix}/@LIB_INSTALL_DIR@
 includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
 
 Name: Ignition @IGN_PROJECT_NAME@
 Description: A set of @IGN_PROJECT_NAME@ classes for robot applications
 Version: @PROJECT_VERSION_FULL@
 Requires:
-Libs: -L${libdir} -l@PROJECT_NAME_LOWER@
-CFlags: -I${includedir}
+Libs:
```

Comment:
Looking into the [.pc file format](http://manpages.ubuntu.com/manpages/focal/man5/pc.5.html) seems like we can remove the empty fields. And that somehow there is an URL field mandatory :roll_eyes:

Reply:
empty fields removed in c20c209d960742e0c71c918cd636de8a09caeb37

continuing to ignore the URL field